deployments/linea-goerli/usdc/migrations/1686918613_configurate_and_ens.ts

```diff
@@ -0,0 +1,250 @@
+    const newMarketObject = { baseSymbol: 'USDC', cometAddress: comet.address };
+    const officialMarketsJSON = JSON.parse(await ENSResolver.text(subdomainHash, ENSTextRecordKey));
+    if (officialMarketsJSON[lineaGoerliChainId]) {
+      officialMarketsJSON[lineaGoerliChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[lineaGoerliChainId] = [newMarketObject];
+    }
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [comet.address, lineaCOMPAddress]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const COMPAmountToBridge = exp(1_000, 18);
+    const USDCAmountToBridge = exp(100_000, 6);
+
+    const goerliActions = [
+      // 1. Set Comet configuration + deployAndUpgradeTo new Comet and set reward config on Linea-Goerli.
+      {
+        contract: lineaMessageService,
+        signature: 'sendMessage(address,uint256,bytes)',
+        args: [bridgeReceiver.address, 0, l2ProposalData]
+      },
+
+      // 2. Approve Goerli's L1StandardBridge to take Timelock's COMP (for bridging)
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1TokenBridge.address, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from Goerli to Linea-Goerli Comet using L1StandardBridge
+      {
+        contract: lineaL1TokenBridge,
+        signature: 'bridgeToken(address,uint256,address)',
+        args: [COMP.address, COMPAmountToBridge, rewards.address]
+      },
+      // 4. Approve Goerli's L1usdcBridge to take Timelock's USDC (for bridging)
+      {
+        contract: USDC,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1usdcBridge.address, USDCAmountToBridge]
+      },
+      // 5. Bridge USDC from Goerli to Linea-Goerli Comet using L1usdcBridge
+      {
+        contract: lineaL1usdcBridge,
+        signature: 'depositTo(uint256,address)',
+        args: [USDCAmountToBridge, rewards.address]
```

Comment:
Just FYI, this was supposed to bridge USDC to Comet instead of the rewards contract, but it was executed like this so I will keep the migration script unchanged.